llvm/include/llvm/Analysis/TargetTransformInfoImpl.h

```diff
@@ -714,6 +714,8 @@ class TargetTransformInfoImplBase {
     switch (ICA.getID()) {
     default:
       break;
+    case Intrinsic::allow_runtime_check:
```

Comment:
This whole function seems to be reinventing isAssumeLikeIntrinsic

Reply:
Not sure, probably all assume like should be there,
but `Intrinsic::objectsize` does not look like it.

I guess missing instruction as parameter prevents use of isAssumeLikeIntrinsic